src/main/java/com/openshift/internal/restclient/model/DeploymentConfig.java

```diff
@@ -164,4 +188,79 @@ public IDeploymentTrigger addTrigger(String type) {
 	public String getDeploymentStrategyType() {
 		return asString(DEPLOYMENTCONFIG_STRATEGY);
 	}
+
+	@Override
+	public boolean haveTriggersFired() {
+		ModelNode causes = get(DEPLOYMENTCONFIG_CAUSES);
+		if (causes == null || causes.getType() != ModelType.LIST)
```

Comment:
The underlying impl is DMR which never returns null; it always returns something.  'null' is not possible but ModelType.UNDEFINED is.